helloWorld.js

```diff
@@ -1,5 +1,9 @@
 console.log("hello world!");
 
+function addTwoNumbers(num1, num2) {
```

Comment:
This seems a bit simple, do we want to add any type checking to ensure that the numbers are actually numbers?